pom.xml

```diff
@@ -184,27 +166,4 @@
             </build>
         </profile>
     </profiles>
-
-    <repositories>
```

Comment:
Why did you removed this repositories? We used by the past it to test Milestone or Relase Candidate versions.

Reply:
I removed them to make sure that everything is using the release version since you only want those by default (I suppose).
For testing, it totally makes sense, I can put it back or we can hide it under a specific profile. What do you think?